front/app/components/admin/SubmitWrapper/index.tsx

```diff
@@ -110,11 +116,20 @@ export default class SubmitWrapper extends PureComponent<Props> {
     this.submitButton = el;
   };
 
+  setSecondaryButtonRef = (el) => {
```

Comment:
Could you add a parameter type here? Or refactor to functional component to mitigate this refactor. :-) Pretty easy component to refactor.